### DISCUSSION TITLE
Smiley Player Trader Release 1.6.1

### DISCUSSION THREAD

**Discussion by Mrcomputer1** [Mrcomputer1/SmileyPlayerTrader#45] [Releases]
# Change Log
* Russian translation update. (By @Fosyaaa on GitHub)
* In the future, the update checker will not notify you about translation only updates that do not update the language you are currently using.

**Note: We recommend backing up your SPT database and configuration file, just in case. While it is unlikely something will go wrong, it is always a good idea to have backup.**

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrcomputer1/SmileyPlayerTrader/releases/tag/v1.6.1'>Smiley Player Trader Release 1.6.1</a>.</em>

**Comment by TheRealFosya** [comment]
What do you think, do SPT needs a UA translation? As far as i know, scince i living in UA, here almost all knows both UA and RU languages, but around 2-3 mills speak and write only on UA language, it's from Lviv, Ternopil' and near this west regions.

**Reply by Mrcomputer1** [reply]
If there is someone willing to contribute a translation for a language, then I think it is worth having it, in case someone, who does want to use SPT, does only know that language or is more comfortable using that language.

According to [bStats](https://bstats.org/plugin/bukkit/SmileyPlayerTrader/6187#location), there are currently no servers (with bStats enabled) running in Ukraine. Although, that doesn't mean that there won't be any in the future or haven't been any in the past. It also is only the server hosting location, so it doesn't show Ukrainian-owned servers hosted outside of Ukraine.

**Reply by TheRealFosya** [reply]
Okay, so not for now.
